### ISSUE TITLE
Open Source Data Labeling | Label Studio

### ISSUE THREAD

**Issue by kinlane** [api-evangelist/inbox#247]
The most flexible data labeling platform to fine-tune LLMs, prepare training data or validate AI models. ~ https://labelstud.io/
